src/interfaces/swig/Features.i

```diff
@@ -308,82 +323,6 @@ namespace shogun
 #endif
 }
 
-/* Templated Class DenseFeatures */
-%include <shogun/features/DenseFeatures.h>
-namespace shogun
-{
-#ifdef USE_BOOL
-	// something wrong with git
-	PROTOCOLS_DENSEFEATURES(BoolFeatures, bool, "?\0", NPY_BOOL)
```

Comment:
this is a long list of code (in swig) that will not be used anymore.....directly editing feature objects in shogun in the style of
```
f = RealFeatures(mat)
f[:,1] = np.array([1,2,3], dtype=np.float64)
```
wont work anymore. But we want to make feature immutable anyways ...